Functions.Templates/Templates-v2/BlobTrigger-Python/template.json

```diff
@@ -0,0 +1,235 @@
+                    "assignTo": "$(CONNECTION_STRING_INPUT)",
+                    "paramId": "blobTrigger-connection",
+                    "required": true,
+                    "defaultValue": "BlobStorageConnectionString"
+                }
+            ],
+            "actions": [
+                "readFileContent_BlueprintFile",
+                "writeFile_BlueprintFile",
+                "readFileContent_BlueprintBody",
+                "appendFileContent_BlueprintBody"
+            ]
+        },
+        {
+            "name": "Add BlobTrigger function to the Blueprint",
+            "type": "AppendToBlueprint", 
+            "inputs" : [
+                {
+                    "assignTo": "$(BLUEPRINT_FILENAME)",
+                    "paramId": "blueprint-existingFileName",
+                    "required": true,
+                    "condition" : {
+                        "name": "ClientId",
+                        "values": [ "VSCode"],
+                        "operator": "IN"
+                    }
+                },
+                {
+                    "assignTo": "$(FUNCTION_NAME_INPUT)",
+                    "paramId": "trigger-functionName",
+                    "required": true,
+                    "defaultValue": "BlobTrigger"
+                },
+                {
+                    "assignTo": "$(PATH_TO_BLOB_INPUT)",
+                    "paramId": "blobTrigger-path",
+                    "required": true,
+                    "defaultValue": "blobname"
+                },
+                {
+                    "assignTo": "$(CONNECTION_STRING_INPUT)",
+                    "paramId": "blobTrigger-connection",
+                    "required": true,
+                    "defaultValue": "BlobStorageConnectionString"
+                }
+            ],
+            "actions": [
+                "readFileContent_BlueprintBody",
+                "appendFileContent_BlueprintBody"
+            ]
+        }
+    ],
+    "actions": [
+        {
+            "name": "readFileContent_FunctionApp",
+            "type": "GetTemplateFileContent",
+            "assignTo": "$(FUNCTION_APP_CONTENT)",
+            "filePath": "function_app.py"
+        },
+        {
+            "name": "writeFile_FunctionApp",
+            "type": "WriteToFile",
+            "source": "$(FUNCTION_APP_CONTENT)",
+            "filePath": "$(APP_FILENAME)",
+            "continueOnError": false,
+            "errorText": "Unable to create the function app",
+            "replaceTokens": true
+        },
+        {
+            "name": "readFileContent_FunctionBody",
+            "type": "GetTemplateFileContent",
+            "assignTo": "$(FUNCTION_BODY)",
+            "filePath" : "function_body.py"
+        },
+        {
+            "name": "appendFileContent_FunctionApp",
+            "type": "AppendToFile",
+            "createIfNotExists" : false,
+            "source": "$(FUNCTION_BODY)",
+            "filePath": "$(SELECTED_FILEPATH)",

```

Comment:
This variable does not exist in input section and so its value would be undefined